site/viamguides/src/install-esp32/install-esp32.md

```diff
@@ -0,0 +1,116 @@
+author: Joyce
+id: install-esp32
+summary: Instructions to install viam-micro-server on ESP32
+categories: Getting-Started, Developer
+environments: web
+status: Published
+feedback link: https://github.com/viam-labs/viamcodelabs/issues
+tags: Getting Started, Developer, Data
+
+# Install viam-micro-server on ESP32
+
+<!-- ------------------------ -->
+
+## Overview
+
+Duration: 1
+
+Learn how to install the `viam-micro-server` on an ESP32 board. It is a lightweight version of `viam-server` which can run on resource-limited embedded systems that cannot run the fully-featured `viam-server`. The `viam-micro-server` is built from the [micro-RDK](https://github.com/viamrobotics/micro-rdk/tree/main).
+
+> aside negative
+> Refer to the [Viam installation guide](https://docs.viam.com/installation/) in the docs for the latest guidelines.
+
+### Prerequisites
+
+- Sign up for a free Viam account, and then [sign in](https://app.viam.com/robots/) to the Viam app
+
+### What You’ll Learn
+
+- How to install the `viam-micro-server` on a microcontroller
+
+### What You’ll Need
+
+- A computer running Mac, Windows, or Linux
+- [ESP32 development board](https://www.amazon.com/FREENOVE-ESP32-WROVER-Compatible-Wireless-Detailed/dp/B0CJJHXD1W) - check the Viam docs for [specific resource requirements](https://docs.viam.com/installation/#compatibility)
+- USB data cable - make sure the cable is capable of data transfer (not just charging)
```

Comment:
Should we link to a resource to help folks learn if their cable is capable of data transfer? Or provide a brief callout of what to look for?

An example resource I found: https://glidedigital.com/how-to-tell-if-a-usb-cable-is-for-charging-or-data-transfer